client/src/modules/journal/journal.js

```diff
@@ -504,8 +505,8 @@ function JournalController(
   function startup() {
     var hasStateFilters = $state.params.filters.length > 0;
     if (hasStateFilters) {
+      $state.params.filters.push({ key : 'includeNonPosted', value : '1' });
```

Comment:
You can just put this in the link the same as the rest of the filters.  There's no reason to put some filters in the controller and others in the links.

Reply:
Like this:
```html
<a data-method="viewInvoice" ui-sref="journal({ filters : [{ key: 'record_uuid', value : row.entity.uuid, displayValue: row.entity.reference}, { key : 'period', value : 'allTime' }, { key : 'includeNonPosted', value : 1, displayValue : 'true' }]})" href>
```